themes-default/slim/src/components/show-header.vue

```diff
@@ -281,6 +284,17 @@ import { api } from '../api';
 import { combineQualities, humanFileSize } from '../utils/core';
 import { AppLink, Asset, QualityPill, StateSwitch } from './helpers';
 
+/**
+ * Return the first item of `values` that is not `null`, `undefined` or `NaN`.
+ * @param {...any} values - Values to check.
+ * @returns {any} - The first item that fits the criteria, `undefined` otherwise.
+ */
+const resolveToValue = (...values) => {
```

Comment:
I know that this **can be** in `./src/utils/core.js`, but for now I don't think we'll need to use it in a lot of places... it's only temporary anyway, as we _probably_ won't be using query params to pass data into this component.

I have a branch that adds that to `utils`, with tests as well, in case we need it.